.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+### Summary
+
+<!-- ✍️ A clear and concise description...-->
+
```

Comment:
related issues/PRs section would be nice

Reply:
done 👍